pkg/builder/builder.go

```diff
@@ -354,9 +355,10 @@ func (b generalBuilder) build(ctx context.Context, pw progresswriter.Writer) err
 
 func (b generalBuilder) checkIfNeedBuild(ctx context.Context) bool {
 	depsFiles := []string{
-		b.PubKeyPath,
 		b.ConfigFilePath,
 	}
+	depsFiles = getDepsFiles(depsFiles)
+	fmt.Println(depsFiles)
```

Comment:
Debugging statement is fine for certain cases, but we should probably switch to structured logging. Filed an issue to discuss separately https://github.com/tensorchord/envd/issues/889

Reply:
> Debugging statement is fine for certain cases, but we should probably switch to structured logging. Filed an issue to discuss separately #889

Good start for team work on debug/logging